packages/ui/src/components/va-alert/VaAlert.vue

```diff
@@ -68,127 +66,110 @@
-      color = this.colorComputed
-    }
-    if (this.border) {
-      color = this.colorComputed
-    }
-    return {
-      color: color,
-    }
-  }
+  props: {
+    color: ({ type: String as PropType<string>, default: 'primary' }),
+    title: ({ type: String as PropType<string>, default: '' }),
+    description: ({ type: String as PropType<string>, default: '' }),
+    icon: ({ type: String as PropType<string>, default: '' }),
+    closeText: ({ type: String as PropType<string>, default: '' }),
+    closeable: ({ type: Boolean as PropType<boolean>, default: false }),
+    dense: ({ type: Boolean as PropType<boolean>, default: false }),
+    outline: ({ type: Boolean as PropType<boolean>, default: false }),
+    center: ({ type: Boolean as PropType<boolean>, default: false }),
+    borderColor: ({ type: String as PropType<string>, default: '' }),
+    border: ({
+      type: String as PropType<string>,
+      default: '',
+      validator: (value: string) => ['top', 'right', 'bottom', 'left', ''].includes(value),
+    }),
+    modelValue: ({
+      type: Boolean as PropType<boolean>,
+      default: true,
+    }),
+  },
+  setup (props, { slots, emit }) {
+    const dark = 'var(--va-dark)'
+
+    const { getColor } = useColors()
+    const colorComputed = computed(() => getColor(props.color))
+
+    const hasIcon = computed(() => props.icon || slots.icon)
+    const hasTitle = computed(() => props.title || slots.title)
+    const borderClass = computed(() => `va-alert__border--${props.border}`)
+    const closeIcon = computed(() => props.closeText || 'close')
+    const alertStyle = computed(() => {
+      let background = colorComputed.value
+      let boxShadow = 'none'
+      if (props.outline) {
+        background = 'transparent'
+      }
+      if (props.border) {
+        background = '#ffffff'
+        boxShadow = 'var(--va-alert-box-shadow)'
+      }
+      return {
+        border: props.outline && `1px solid ${colorComputed.value}`,
+        padding: props.dense && '0.25rem 0.75rem',
+        background,
+        boxShadow,
+      }
+    })
+    const contentStyle = computed(() => {
+      let color = getTextColor(colorComputed.value)
+      if (props.outline) {
+        color = colorComputed.value
+      }
+      if (props.border) {
+        color = dark
+      }
+      return {
+        alignItems: props.center && 'center',
+        color,
+      }
+    })
+    const titleStyle = computed(() => {
+      let color = getTextColor(colorComputed.value)
+      if (props.outline) {
+        color = colorComputed.value
+      }
+      if (props.border) {
+        color = colorComputed.value
+      }
+      return { color }
+    })
+    const borderStyle = computed(() => ({
+      backgroundColor: props.borderColor
+        ? getColor(props.borderColor)
+        : colorComputed.value,
+    }))
 
-  get borderClass () {
-    return `va-alert__border--${this.border}`
-  }
+    const { valueComputed } = useStateful(props, emit)
```

Comment:
valueComputed has not been returned in setup function.

Reply:
Fixed